components/developer/gdb/gdb.p5m

```diff
@@ -34,9 +38,21 @@ set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
 set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
-file path=usr/bin/gdb
+
+file path=usr/bin/gcore
+# The dependency resolution for python 3.9 does not work as expected. For one the searchpath is different 
+# but even with adapting the searchpath to the correct directories pkg is unable to find the library while ldd
+# and ld.so are able to. This is a bug in pkg so we workaround it for now.
+file path=usr/bin/gdb pkg.depend.bypass-generate=.*/libpython3.9.so.*
 file path=usr/bin/gdbtui mode=0555
+file path=usr/include/ansidecl.h
+file path=usr/include/bfd.h
+file path=usr/include/bfdlink.h
+file path=usr/include/dis-asm.h
 file path=usr/include/gdb/jit-reader.h
+file path=usr/include/plugin-api.h
+file path=usr/include/symcat.h
+file path=usr/lib/$(MACH64)/charset.alias
```

Comment:
Will this appear in the published package?  It shouldn't.

Reply:
Ok, removed the .alias file